swap_meet/clothing.py

```diff
@@ -1,2 +1,55 @@
-class Clothing:
-    pass
\ No newline at end of file
+import uuid
+from swap_meet.item import Item
+
+# Wave 5- SJ
+# 1. Clothing class has 3 attributes, id, condition and fabric
+# 1a. id attribute is by default a unique integer
+# 1b. condition attribute by default, set to 0
+# 1c. fabric attribute is by default the string "Unknown"
+# 1d. Clothing class can inherit id and condition from Item class
+#     Clothing class shares the same id and condition attributes as Item class
+#     Item constructor: def __init__(self, id=None, condition = 0):
+#     class ExampleChildClass(ExampleParentClass):
+
+
+
+class Clothing(Item):
+    
+    def __init__(self, id=None, condition = 0, fabric = "Unknown"):
+        super().__init__(id, condition)
```

Comment:
Nice work with the `super` function, before using this we know that to get the same behavior we would have to write a bunch of the same code, which doesn't fall in line with our `DRY` paradigm, this saves us a lot of typing and causes the code base to much more concise. Also, great work on passing in the `None` default param in the `__init__` function.